### DISCUSSION TITLE
FleatherField Insert Image

### DISCUSSION THREAD

**Discussion by OlaveraLuffy** [fleather-editor/fleather#196] [Q&A] [answered]
Hi! I am trying to insert an image in the fleatherfield via an upload button that opens up the gallery and can select an image. How do I incorporate the image inside the fleatherfield? As seen in the image.

Also, does fleather have an increase and decrease font size in the fleathertoolbar.basic? Thank you for any response! <3

![1](https://github.com/fleather-editor/fleather/assets/38834838/b2ea67c1-251f-4798-92c4-19d6c2979cbc)

**Comment by amantoux** [comment, marked as answer]
Hi @OlaveraLuffy , sorry for the late reply
You can use an `Embed` block (see corresponding attributes [here](https://fleather-editor.github.io/docs/getting-started/concepts/attributes/))

In the package example, you can find a example of a builder to render the image
https://github.com/fleather-editor/fleather/blob/cc0ea767b6a15e6b3763d8d286b40756454f1054/packages/fleather/example/lib/main.dart#L114)

**Reply by OlaveraLuffy** [reply]
Hi! Sorry for the late reply. Can you show me how to integrate the example properly in my code? Basically, I want the _selectedImage to be put inside the fleather field. Not outside, As you can see in the example image

![image](https://github.com/fleather-editor/fleather/assets/38834838/d4034ded-0fb6-4b58-a7d4-042adc979b3b)


![image](https://github.com/fleather-editor/fleather/assets/38834838/12dae70f-b012-4330-9a33-34685c58f322)
